src/network/getChainPublicClient.ts

```diff
@@ -0,0 +1,23 @@
+import { createPublicClient, http } from 'viem';
+import { type Chain, base, baseSepolia, mainnet, sepolia } from 'viem/chains';
+
+export const baseChainsIds: Chain['id'][] = [base.id, baseSepolia.id];
+export const ensUniversalResolverChainIds: Chain['id'][] = [
```

Comment:
I don't follow, what's the difference of this vs `chainsById` ? It seems is just the `key` of chainsById